cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/GroundConversionTest.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package com.powsybl.cgmes.conversion.test;
+
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.VoltageLevel;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * @author Sophie Frasnedo {@literal <sophie.frasnedo at rte-france.com>}
+ */
+
+class GroundConversionTest {
+
+    @Test
+    void groundConversionTest() {
+        Network network = Network.read("groundTest.xml", getClass().getResourceAsStream("/groundTest.xml"));
+        long groundCount = network.getVoltageLevelStream().map(VoltageLevel::getGrounds).count();
+        assertEquals(2, groundCount);
```

Comment:
You can call `network.getGroundCount()` to simplify the test.

Also, the followings can also be tested:
- the id and names of the grounds are imported (`GroundAdderImpl` should be modified to use the names);
- the grounds are attached to the right voltage levels ("CV" and "OU" are both attached to voltage level "S");
- the grounds' terminals are not null and not connected (both ground disconnectors are open).

Reply:
Ok, finally @zamarrenolm do not reduce the file, it is okay like this.